src/indicators_1d.jl

```diff
@@ -0,0 +1,225 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+@muladd begin
+#! format: noindent
+
+# this method is used when the indicator is constructed as for shock-capturing volume integrals
+# empty cache is default
+function create_cache(::Type{<:IndicatorNeuralNetwork},
```

Comment:
```suggestion
function Trixi.create_cache(::Type{<:IndicatorNeuralNetwork},
```

Reply:
No. It's never used from within Trixi.jl, only from the `IndicatorNeuralNetwork` ctors:
https://github.com/trixi-framework/TrixiSmartShockFinder.jl/pull/1/files#diff-e52d4e73367bdb8c0399e730758638721a843b45793f67374c21145d4abfc2c5R66
or
https://github.com/trixi-framework/TrixiSmartShockFinder.jl/pull/1/files#diff-e52d4e73367bdb8c0399e730758638721a843b45793f67374c21145d4abfc2c5R87
Otherwise I'd have to change the call there to `Trixi.create_cache` as well, but I don't feel like we should improve this package more than necessary. What do you think?

Reply:
Ok, if you're sure it works correctly, we can leave it as it is

Reply:
I actually did exactly what you suggested a few days ago in https://github.com/trixi-framework/TrixiSmartShockFinder.jl/commit/fb91bd98193de63b8e19efe93853d87bdc848545 and realized then that it doesn't work 😅. Thus I'd leave it as it is.